docs/manuals/core/config/data-node-config.md

```diff
@@ -346,3 +346,43 @@ In this example, we configure an in_memory data node with the id "date", the sco
     the method `Config.configure_data_node()^` with parameter `storage_type="in_memory"`.
 
 [:material-arrow-right: The next section introduces the task configuration](task-config.md).
+
+# JSON
+
+An `JSONDataNode^` is a specific data node used to model JSON
+file data. To add a new _JSON_ data node configuration, the `Config.configure_json_data_node_node()^` method can be used.
+In addition to the generic parameters described in the previous section
+[Data node configuration](data-node-config.md), a mandatory and an optional parameters can be provided.
+
+-   The _**path**_ is a mandatory parameter that represents the JSON file path used by Taipy to read and write the data.
+
+-   When the _**exposed_type**_ is given as a parameter, if the provided value is a custom class, data node will create a custom object of that class if the JSON is an object, or a list of custom objects if the JSON data is a list. If _exposed_type_ is not provided, the data node will treat the JSON object as a dictionary.
```

Comment:
"When the _**exposed_type**_ is given as a parameter, if the provided value is a custom class, data node"
->
"If _**exposed_type**_ is set to a custom class then the data node"